src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -210,7 +210,7 @@ function ExportAnnotations({
     >
       {exportableAnnotations.length > 0 ? (
         <>
-          <p className="text-color-text-light mb-3">
```

Comment:
This was needed only because the label previously placed under it was missing a `block` class.